cli/commands/delete_lease.go

```diff
@@ -0,0 +1,37 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"github.com/codegangsta/cli"
+	"github.com/deis/k8s-claimer/htp"
+)
+
+// DeleteLease is a cli.Command action for deleting a lease
+func DeleteLease(c *cli.Context) {
+	server := c.GlobalString("server")
+	if server == "" {
+		log.Fatalf("Server missing")
+	}
+	if len(c.Args()) < 1 {
+		log.Fatalf("Lease token missing")
+	}
+	leaseToken := c.Args()[0]
```

Comment:
@vdice can we see how it works by just using the cmdline param for now, and look for the env var if it turns out to be more convenient later?